quinn-proto/src/congestion.rs

```diff
@@ -12,6 +12,8 @@ pub use bbr::{Bbr, BbrConfig};
 pub use cubic::{Cubic, CubicConfig};
 pub use new_reno::{NewReno, NewRenoConfig};
 
+const BASE_DATAGRAM_SIZE: u64 = 1200;
```

Comment:
Style nit: we usually put our constants near the bottom of the module, not near the top.